### ISSUE TITLE
org.eclipse.rdf4j.http.server.repository.RepositoryListController is reporting all repositories as writeable

### ISSUE THREAD

**Issue by asotona** [eclipse/rdf4j#1062]
org.eclipse.rdf4j.http.server.repository.RepositoryListController is always reporting all repositories as writeable, instead of asking the underlying SAIL. See line RepositoryListController, line 80:
```
72:				repositoryManager.getAllRepositoryInfos(false).forEach(info -> {
73:					QueryBindingSet bindings = new QueryBindingSet();
74:					bindings.addBinding("uri", vf.createIRI(namespace, info.getId()));
75:					bindings.addBinding("id", vf.createLiteral(info.getId()));
76:					if (info.getDescription() != null) {
77:						bindings.addBinding("title", vf.createLiteral(info.getDescription()));
78:					}
79:					bindings.addBinding("readable", vf.createLiteral(true));
80:					bindings.addBinding("writable", vf.createLiteral(true));
81:					bindingSets.add(bindings);
82:				});
```

**Comment by jeenbroekstra** [comment]
Somewhat related is that the `LocalRepositoryManager` object also just hardcodes this value to `true` when creating the `RepositoryInfo`.

**Comment by jeenbroekstra** [comment]
This looks like something we could include in a patch release. I'll reschedule and cherrypick to the correct branch.

**Comment by jeenbroekstra** [comment]
@barthanssens I'm sorry, I may be jumping the gun here a bit. Do I understand correctly that the fix you did for this is only part of the solution and to fully address the problem we'll need a fix for #1194 as well? If that's so, perhaps it's pointless to add this to the patch release (at least not until we've also fixed the followup issue).

**Comment by barthanssens** [comment]
@jeenbroekstra I'll have to double check, I was under the impression that it would be fixed for remote repositories, but not for local repositories
